--- ncspot-0.11.2/src/main.rs	2022-10-15 19:48:44.000000000 +0900
+++ ncspot-0.11.2/src/main.rs.new	2022-10-16 15:07:11.186159584 +0900
@@ -168,6 +168,7 @@
         credentials = credentials_prompt(Some(error_msg))?;
     }
 
+    println!("Connecting to Spotify..");
     let backend = cursive::backends::try_default().map_err(|e| e.to_string())?;
     let buffered_backend = Box::new(cursive_buffered_backend::BufferedBackend::new(backend));
 
@@ -179,7 +180,6 @@
 
     let event_manager = EventManager::new(cursive.cb_sink().clone());
 
-    println!("Connecting to Spotify..");
     let spotify = spotify::Spotify::new(event_manager.clone(), credentials, cfg.clone());
 
     let library = Arc::new(Library::new(&event_manager, spotify.clone(), cfg.clone()));
